file_paths.md

```diff
@@ -163,160 +163,195 @@ The package will create the following services `puppet`, `mcollective`,
 and `pxp-agent`, all running as `LocalSystem` by default. It will not
 create a `puppet` user or group.
 
-    Path                                      Setting                        3.x
-    C:\ProgramData                                                           n/a
+    Path                                          Setting                        3.x
+    C:\ProgramData\PuppetLabs *                                                  n/a
```

Comment:
I couldn't get vanagon to build with spaces in the path name. So, from what I last saw in the vanagon build, it was building under `PuppetLabs`.